debian/scripts/usr/local/bin/config_save

```diff
@@ -0,0 +1,36 @@
+# Save current configurations to partition containing conf.d
+#
+# Copyright (C) 2018 Benedict Lau <benedict.lau@groundupworks.com>
+
+CONFDIR=conf.d
+
+try_partition() {
+    # Create file system node from partition
+    #mknod /dev/$1 b $2 $3 2>/dev/null
+
+    # Mount partition as read-write
+    mount /dev/$1 /mnt
+
+    # Find conf.d in root of partition and save current configurations in /etc
+    if [ -d "/mnt/$CONFDIR" ]; then
+        echo Saving configurations to /dev/$1: "/mnt/$CONFDIR/50-node-config-save.tar.gz"
+        # TODO Handle symlinks properly (i.e. excluding them from the tar)
+        tar --create --gzip -f "/mnt/$CONFDIR/50-node-config-save.tar.gz" -C /etc .
+    fi
+
+    # Unmount partition
+    umount /mnt
+}
+
+# Mount proc if not already mounted
+#mount -t proc proc /proc
+
+cat /proc/partitions | while read major minor size name; do
```

Comment:
This writes to multiple partitions if they all have `conf.d`. We don't expect that to happen and it's probably not a big deal.

Reply:
I dunno - I'd be wanting it to only save to one of them - perhaps exit after the first found.  Perhaps allowing the device name to be given as an optional parameter

Reply:
I added breaking after first found